src/app/(settings)/settings/profile/components/Profile.tsx

```diff
@@ -90,4 +92,4 @@ const Profile = ({ loginUser }: SettingsProps) => {
   );
 };
 
-export default Profile;
\ No newline at end of file
+export default Profile;
```

Comment:
For non-pages file, I always prefer named export instead of default export.

Reply:
Sure, Fixed